### PR TITLE
Format C++ tests with Google style

### PR DESCRIPTION
**Pull request by y-guyon**
Add tests/gtest/.clang-format for that purpose.
Fix .github/workflows/clang-format-check.yml

### CODE REVIEW

File: tests/gtest/avifincrtest.cc [hunk 1]
```diff
@@ -16,86 +15,97 @@ using testing::Bool;
...
-namespace
-{
+namespace libavif {
+namespace {
 
 //------------------------------------------------------------------------------
 
 // Used to pass the data folder path to the GoogleTest suites.
-const char * dataPath = nullptr;
+const char *dataPath = nullptr;
```
Comment (jzern) [inline tests/gtest/avifincrtest.cc:24]: more common would be 'const char* dataPath'. The tool just took a guess here, but it's inconsistent with the formatting in tests/gtest/avifincrtest_helpers.h for example

Comment (jzern) [reply tests/gtest/avifincrtest.cc:24]: As a one off you could force the option one way or another, then the tool should keep the files in sync

File: tests/gtest/.clang-format [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+Language: Cpp
+BasedOnStyle: Google
+DerivePointerAlignment: false
```
Comment (wantehchang) [inline tests/gtest/.clang-format:4]: The example in https://clang.llvm.org/docs/ClangFormatStyleOptions.html also sets `PointerAlignment` to `Left`:

```
Language: Cpp
# Force pointers to the type for C++.
DerivePointerAlignment: false
PointerAlignment: Left
```

**Review by wantehchang — CHANGES_REQUESTED** [review]
This looks good. We should also follow the naming conventions of the Google C++ Style Guide. I assume you are planning to rename the functions and variables in a separate pull request.

File: tests/gtest/.clang-format (hunk 2 above)
Comment (y-guyon) [reply tests/gtest/.clang-format:4]: Done but I believe `PointerAlignment: Left` is set by `BasedOnStyle: Google` already.

**Comment by y-guyon** [comment]
> This looks good. We should also follow the naming conventions of the Google C++ Style Guide. I assume you are planning to rename the functions and variables in a separate pull request.

Yes.

**Review by vrabaud — APPROVED** [review]

File: tests/gtest/.clang-format (hunk 2 above)
Comment (wantehchang) [reply tests/gtest/.clang-format:4]: You are right. Would you like to remove `PointerAlignment: Left`?

**Pull request merged**